Vagrantfile

```diff
@@ -135,7 +135,7 @@ end
 
 # Clone Spryker (if repository is given)
 if defined?(SPRYKER_REPOSITORY)
-  if not Dir.exists?(SPRYKER_DIRECTORY) and not SPRYKER_REPOSITORY.empty?
+  if Dir[SPRYKER_DIRECTORY + "/*"].empty? and not SPRYKER_REPOSITORY.empty?
```

Comment:
Please also check for the dotfiles - Dir doesn't return dotfiles in array, even if they are present, which can make it fail if there's some leftover files. To reproduce:
```
➜  ~ mkdir t13
➜  ~ touch t13/.gitignore
➜  ~ irb
2.2.1 :001 > Dir['t13/*']
 => []
➜  ~ git clone https://github.com/spryker/devvm t13
fatal: destination path 't13' already exists and is not an empty directory.
```